src/server/api/user/index.ts

```diff
@@ -30,6 +32,9 @@ export const UserApi = {
     express.get(ApiEndPoint.User.invitationAccept(), acceptInvitation)
     express.get(ApiEndPoint.User.invitationSendEmail(), AuthMiddleware.requireEditUser, sendInvitationEmail)
 
+    express.post(ApiEndPoint.User.demote(), AuthMiddleware.requireAdmin, demoteToUser)
+    express.post(ApiEndPoint.User.promote(), AuthMiddleware.requireAdmin, promoteToAdmin)
```

Comment:
why the need of these two endpoints? couldn't you call updateUserRoles only ?

Reply:
because I would like to remove, in the near future, the admin role and manage this information in a different way because the management of the userroles begins to be complicated